lib/src/lib.rs

```diff
@@ -0,0 +1,345 @@
+use anyhow::{anyhow, Result};
+
+#[macro_use]
+extern crate log;
+
+extern crate strum;
+#[macro_use]
+extern crate strum_macros;
+
+mod core;
+use crate::core::verifier::policy::opa;
+
+/// The supported TEE types:
+/// - TDX: TDX TEE.
+/// - SGX: SGX TEE.
+/// - SEVSNP: SEV-SNP TEE.
+/// - SAMPLE: A dummy TEE that used to test/demo the attestation service functionalities.
+#[derive(Debug, EnumString)]
+pub enum TEE {
```

Comment:
Because the TEE enum needs to be exported by this Crate:
```
use attestation_service::{Service, TEE};

#[tokio::main]
async fn main() {
    let service = Service::new().unwrap();
    let res = service.default_policy(TEE::SAMPLE).await;
    assert!(res.is_ok());
 }
```
But the TEE enum's implementation is internal.